lib/connectors/memory.js

```diff
@@ -567,9 +567,9 @@ function applyFilter(filter) {
         return compare(example.neq, value) !== 0;
       }
 
-      if ('between' in example ) {
-        return ( testInEquality({gte:example.between[0]}, value) &&
-                 testInEquality({lte:example.between[1]}, value) );
+      if ('between' in example) {
+        return (testInEquality({ gte:example.between[0] }, value) &&
+                 testInEquality({ lte:example.between[1] }, value));
```

Comment:
@bajtos can field this one. Please update the style-guide too with this answer as I'm also curious.